site/app/views/grading/ElectronicGraderView.php

```diff
@@ -82,6 +101,8 @@ public function overviewPage($gradeable, $sections) {
     </div>
 </div>
 HTML;
+        }
+        }
```

Comment:
Fix the indentation on this curly brace so it aligns with what ever opened it. That way, we can scroll up in the file and find the associated if or else or whatever that opened it.